docker-compose-local.yaml

```diff
@@ -13,8 +16,13 @@ services:
     build:
       context: .
       dockerfile: Dockerfile
-    command: ["proxy", "--mlmd-hostname", "localhost", "--mlmd-port", "9090"]
+      network: host
```

Comment:
I'm not sure we need this `network: host` 🤔

Reply:
it's used only during the build of the container and it was needed on my machine with podman to successfully build the image.